src/components/Breadcrumb/Breadcrumb.vue

```diff
@@ -228,12 +226,21 @@ export default {
 		}
 	}
 
-	&::before {
+	&::after {
+		content: '';
 		display: flex;
 		align-items: center;
-		order: 1;
 		color: var(--color-border-dark);
 		font-size: 26px;
+		width: 8px;
```

Comment:
```suggestion
		width: 8px;
		min-width: 8px;
```

Reply:
Seems to fix https://github.com/nextcloud/nextcloud-vue/pull/1816#issuecomment-812667792